scripts/check-schema-and-codegen/utils.ts

```diff
@@ -64,6 +47,7 @@ export const checkIsAncestor = async (commit: string): Promise<boolean> => {
     return true;
   } catch (error) {
     process.chdir(originalDir);
+    // Error status 1 and 128 means that the commit is not an anecestor and the user must fetch.
```

Comment:
I discussed this offline but would love to see a reference to this here. Just so its easy to differentiate between what 1 and 128 means.